highfive/configs/rust-lang/rust.json

```diff
@@ -188,6 +193,10 @@
         "src/tools/miri": {
             "message": "Some changes occured to the Miri submodule",
             "reviewers": ["@rust-lang/miri"]
+        },
+        "library": {
+            "message": "Thank you for submitting a new PR for the library teams! If this PR contains a stabilization of a library feature that has not already completed FCP in its tracking issue, introduces new or changes existing unstable library APIs, or changes our public documentation in ways that create new stability guarantees for specific behavior then please comment with `r? rust-lang/libs-api @rustbot label +T-libs-api` to request review from a libs-api reviwer.",
+            "reviewers": []
```

Comment:
I double checked impl for highfive to make sure it would correctly format this message even if there are no reviewers. Seems like it's all fine and doesn't do much checking on the content of the `mentions` body until it gets to here:

* https://github.com/rust-lang/highfive/blob/master/highfive/newpr.py#L375-L380
* https://github.com/rust-lang/highfive/blob/master/highfive/newpr.py#L136

The one problem I did find is that it will print the `cc` unconditionally, so if we don't put anything in this mentions field won't look as nice.

```python
>>> "%scc %s" % ("hey thanks for the PR!\n\n", ','.join([]))
'hey thanks for the PR!\n\ncc '
```

I can change the impl to branch on if the reviewers list is empty and to only print the message if so, but I don't want to jump right to that since I feel like that might be an abuse of the "mentions" field and I don't want to turn this into a bunch of spaghetti.

Reply:
I think it makes sense to slightly adjust the impl as you suggest, and I would also add an explicit "no worries if you're not sure - the reviewer will take a look and make a decision to forward on if necessary"

Reply:
I agree that this is a minor "abuse" but it'll get cleanly solved once we migrate to triagebot doing this, for now it's ok.